pkg/activator/throttler.go

```diff
@@ -142,7 +170,7 @@ func (t *Throttler) updateCapacity(revision *v1alpha1.Revision, breaker *queue.B
 		// The concurrency is unlimited, thus hand out as many tokens as we can in this breaker.
 		targetCapacity = t.breakerParams.MaxConcurrency
 	}
-
+	targetCapacity = targetCapacity / t.getActivatorCount()
```

Comment:
We "lose" some endpoints if the number is not evenly divisible. Do we care?

Reply:
The activator queue will buffer the request that exceeds the `targetCapacity`. Yes we do lose endpoints but we are not losing the request so its not huge concern IMO.